dot15d4-frame/src/ie/nested.rs

```diff
@@ -886,13 +897,32 @@ impl<T: AsRef<[u8]>> TschSlotframeAndLink<T> {
             &self.data.as_ref()[1..],
         )
     }
+
+    /// Return the length of TSCH Slotframe and Link IE in bytes.
+    #[allow(clippy::len_without_is_empty)]
+    pub fn len(&self) -> usize {
+        let slotframe_descriptors_len =
+            self.slotframe_descriptors().map(|d| d.len()).sum::<usize>();
+        1 + slotframe_descriptors_len
+    }
+
+    /// Return the content of the TSCH slotframe and link IE.
+    pub fn content(&self) -> &[u8] {
+        // TODO: check if we should set end index
+        &self.data.as_ref()[1..self.len()]
+    }
 }
 
 impl<T: AsRef<[u8]> + AsMut<[u8]>> TschSlotframeAndLink<T> {
     /// Set the number of slotframes field.
     pub fn set_number_of_slotframes(&mut self, number_of_slotframes: u8) {
         self.data.as_mut()[0] = number_of_slotframes;
     }
+
+    /// Return the content of the TSCH slotframe and link IE.
+    pub fn content_mut(&mut self) -> &mut [u8] {
+        &mut self.data.as_mut()[1..]
+    }
```

Comment:
I think `content_mut` should be implemented for `T: AsRef<[u8]> + AsMut<[u8]> + ?Sized`. Can you take a look at an implementation of `payload_mut`.